docs/topics/readme/caching-README.adoc

```diff
@@ -0,0 +1,82 @@
+= ${mission} - ${runtime} Booster
+
+[IMPORTANT]
+====
+While you can run and interact with this booster on localhost, it requires that you also have a caching server installed and configured. This booster runs best when deployed on OpenShift with a JBoss Data Grid caching server. 
+For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:${guideURL}[${runtime} Runtime Guide].
+====
+
+${prerequisite}
+
+${cicdSection}
+
+== Running the Booster on a Single-node OpenShift Cluster
+If you have a single-node OpenShift cluster, such as Minishift or Red Hat Container Development Kit, link:http://appdev.openshift.io/docs/minishift-installation.html[installed and running], you can also deploy your booster there. A single-node OpenShift cluster provides you with access to a cloud environment that is similar to a production environment.
+
+To deploy your booster to a running single-node OpenShift cluster:
+
+. Navigate to the root directory of your booster.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ ${gitCMD}
+
+$ cd ${targetRepository}
+----
+
+. Log in and create your project.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login -u developer -p developer
+
+$ oc new-project MY_PROJECT_NAME
+----
+
+. Deploy the caching service.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc create -f service.cache.yaml
+----
+
+. Deploy your booster.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ ${OSORunCMD}
+----
+
+
+
+== Interacting with the Booster on a Single-node OpenShift Cluster
+
+. Determine the URL of the `greeting` service.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get route ${app-name} -o jsonpath={$.spec.host}
+
+${app-name}-MY_PROJECT_NAME.LOCAL_OPENSHIFT_HOSTNAME
+----
+
+. Navigate to the `greeting` service in your browser.
+. Click _Greet_ once.
++
+Notice the `duration` value is above `2000`. Also notice _Cache state_ has changed form `Not cached` to `Cached`.
```

Comment:
In a different file, you have

> Notice the `duration` value is above `2000`. Also notice the cache state has changed form `No cached value` to `A value is cached`.

Which one is it?

Reply:
thank you @Ladicek forgot to update this section with new instructions.